model/version.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package model
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	VERSION_MAJOR = 0
+	VERSION_MINOR = 8
+	VERSION_PATCH = 0
+	BUILD_NUMBER  = "_BUILD_NUMBER_"
+	BUILD_DATE    = "_BUILD_DATE_"
+)
+
+func GetFullVersion() string {
+	return fmt.Sprintf("%v.%v.%v", VERSION_MAJOR, VERSION_MINOR, VERSION_PATCH)
```

Comment:
Build number is included in the UI, logs, and when you run -version on the cmd line.  Its just not part of the "official" build numbering.